sdk/textanalytics/ai-text-analytics/package.json

```diff
@@ -120,6 +120,7 @@
     "karma-mocha": "^2.0.1",
     "karma-mocha-reporter": "^2.2.5",
     "karma-remap-istanbul": "^0.6.0",
+    "karma-source-map-support": "~1.4.0",
```

Comment:
Not that it's wrong, but how'd this change sneak through?

Reply:
reverted